src/ImageSharp/Common/Helpers/BitOperations.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Runtime.CompilerServices;
+
+#if !NETCOREAPP3_1
```

Comment:
```suggestion
#if !NETCOREAPP3_1_OR_GREATER
```
is this macro available here?

Otherwise it's easy to miss if new targets are enabled. 
E.g. for .NET 6.0 this code would be hit, instead the one from the built-in BitOperations.

Reply:
Yes, it is defined. So, used it instead.